### PR TITLE
Edit docs/learn/wallets/firefly-wallet.md

### PR DESCRIPTION
**Pull request by DafPhil**

### CODE REVIEW

File: docs/learn/wallets/firefly-wallet.md [hunk 1]
```diff
@@ -76,139 +81,133 @@ The only thing you need to move your Tokens to the new network is your current I
...
+Wallets contain a collection of addresses and those addresses are grouped under this wallet to help users keep their addresses organized.
 
-Important to mention here: Users can only set up further Wallets if the first Wallet has already funds in it. So if Users want to set up 5 different Wallets they need a small amount of IOTA in every newly created Wallet before another one can be created.
+It is important to mention here that users can only set up further wallets if the first wallet already contains funds. If users want to set up five different wallets, they need a small amount of IOTA in every newly created wallet before another one can be created.
 
-### Wallet Detail view
+### Wallet Detail View
 
-[![wallet detail view](/img/learn/firefly/wallet_detail_view.png)](/img/learn/firefly/wallet_detail_view.png)
+![wallet detail view](/img/learn/firefly/wallet_detail_view.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:121]:
```suggestion
[![wallet detail view](/img/learn/firefly/wallet_detail_view.png)](/img/learn/firefly/wallet_detail_view.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 2]
```diff
@@ -76,139 +81,133 @@ The only thing you need to move your Tokens to the new network is your current I
...
 
 **Hide Wallet**
 
--   Empty Wallets can be hidden to keep your Main Wallet overview organized. If a wallet still contains funds you will not be able to hide it by clicking this option but will be offered to transfer the funds to another Wallet and hide the Wallet afterward. A hidden Wallet can be shown again by enabling the function "show hidden wallets" in the advanced account settings.
+- Empty wallets can be hidden to keep your main wallet overview organized. If a wallet still contains funds you will not be able to hide it by clicking this option but will be offered to transfer the funds to another wallet and hide the wallet afterward. A hidden wallet can be shown again by enabling the function "show hidden wallets" in the advanced account settings.
 
 #### Transaction detail view.
 
-[![transaction details view](/img/learn/firefly/transaction_details_view.png)](/img/learn/firefly/transaction_details_view.png)
+![transaction details view](/img/learn/firefly/transaction_details_view.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:147]:
```suggestion
[![transaction details view](/img/learn/firefly/transaction_details_view.png)](/img/learn/firefly/transaction_details_view.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 3]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
 
 ### Creating a new Ledger profile
 
-Creating a new Ledger profile in Firefly is very quick and simple. Once you have downloaded Firefly, open the app and follow the setup instructions. You first need to review and accept the terms and conditions, set a theme (dark or light), and choose a profile name. Select "Create a new wallet".
+Creating a new Ledger profile in Firefly is quick and simple. Once you have downloaded Firefly, open the app and follow the setup instructions. You first need to review and accept the terms and conditions, set a theme (dark or light), and choose a profile name. Select "Create a new wallet".
 
-[![setup a wallet ledger](/img/learn/firefly/setup_a_wallet_ledger.png)](/img/learn/firefly/setup_a_wallet_ledger.png)
+![setup a wallet ledger](/img/learn/firefly/setup_a_wallet_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:234]:
```suggestion
[![setup a wallet ledger](/img/learn/firefly/setup_a_wallet_ledger.png)](/img/learn/firefly/setup_a_wallet_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 4]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Press "I want a hardware wallet" to begin the Ledger profile setup process.
 
-[![create a wallet ledger](/img/learn/firefly/create_a_wallet_ledger.png)](/img/learn/firefly/create_a_wallet_ledger.png)
+![create a wallet ledger](/img/learn/firefly/create_a_wallet_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:238]:
```suggestion
[![create a wallet ledger](/img/learn/firefly/create_a_wallet_ledger.png)](/img/learn/firefly/create_a_wallet_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 5]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 On the next page, you will be asked to set a PIN code. This PIN is used to log in to your wallet and blocks other people from logging in and viewing your balance transaction history. Choose a PIN that only you know. Re-enter your PIN on the next page to confirm.
 
-[![set pin ledger](/img/learn/firefly/set_pin_ledger.png)](/img/learn/firefly/set_pin_ledger.png)
+![set pin ledger](/img/learn/firefly/set_pin_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:242]:
```suggestion
[![set pin ledger](/img/learn/firefly/set_pin_ledger.png)](/img/learn/firefly/set_pin_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 6]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Now, Firefly will check that your Ledger Nano is connected. Make sure the IOTA app is open on your device and that Ledger Live is closed. If you have connection problems follow the "Tips if your Ledger isn't connecting" guide in Firefly.
 
-[![connect ledger ledger](/img/learn/firefly/connect_ledger.png)](/img/learn/firefly/connect_ledger.png)
+![connect ledger ledger](/img/learn/firefly/connect_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:246]:
```suggestion
[![connect ledger ledger](/img/learn/firefly/connect_ledger.png)](/img/learn/firefly/connect_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 7]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Hit "Continue", and that's all there is to it. It really is that simple. You can now enter the dashboard and begin exploring Firefly.
 
-[![setup complete ledger](/img/learn/firefly/setup_complete_ledger.png)](/img/learn/firefly/setup_complete_ledger.png)
+![setup complete ledger](/img/learn/firefly/setup_complete_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:250]:
```suggestion
[![setup complete ledger](/img/learn/firefly/setup_complete_ledger.png)](/img/learn/firefly/setup_complete_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 8]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 ### Receiving tokens
 
 If you would like to receive funds to your new Ledger profile you can press the "Receive" button on the dashboard.
 
-[![generate address ledger](/img/learn/firefly/generate_address_ledger.png)](/img/learn/firefly/generate_address_ledger.png)
+![generate address ledger](/img/learn/firefly/generate_address_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:256]:
```suggestion
[![generate address ledger](/img/learn/firefly/generate_address_ledger.png)](/img/learn/firefly/generate_address_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 9]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 On this page you can select "Generate address" and Firefly will prompt you to confirm the address matches the one displayed on your Ledger device.
 
-[![confirm receive address ledger](/img/learn/firefly/confirm_receive_address_ledger.png)](/img/learn/firefly/confirm_receive_address_ledger.png)
+![confirm receive address ledger](/img/learn/firefly/confirm_receive_address_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:260]:
```suggestion
[![confirm receive address ledger](/img/learn/firefly/confirm_receive_address_ledger.png)](/img/learn/firefly/confirm_receive_address_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 10]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Scroll through the address on your Ledger device. Confirm it matches by pressing both buttons when your Ledger reads "Ok".
 
 Note: This is a security check to ensure that you are connected to the official, secure version of Firefly. It ensures that no malicious software can alter the address you receive to and that it matches the one generated by your Ledger.
 
-[![display ledger address](/img/learn/firefly/display_ledger_address.png)](/img/learn/firefly/display_ledger_address.png) [![okay ledger display](/img/learn/firefly/okay_ledger_display.png)](/img/learn/firefly/okay_ledger_display.png)
+![display ledger address](/img/learn/firefly/display_ledger_address.png)![okay ledger display](/img/learn/firefly/okay_ledger_display.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:266]:
```suggestion
[![display ledger address](/img/learn/firefly/display_ledger_address.png)](/img/learn/firefly/display_ledger_address.png) [![okay ledger display](/img/learn/firefly/okay_ledger_display.png)](/img/learn/firefly/okay_ledger_display.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 11]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 You can now copy the address and share it with a friend or exchange from which you wish to receive tokens. Always make sure the address is the same as the one displayed on your Ledger device.
 
-[![wallet view ledger](/img/learn/firefly/wallet_view_ledger.png)](/img/learn/firefly/wallet_view_ledger.png)
+![wallet view ledger](/img/learn/firefly/wallet_view_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:270]:
```suggestion
[![wallet view ledger](/img/learn/firefly/wallet_view_ledger.png)](/img/learn/firefly/wallet_view_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 12]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 ### Sending tokens
 
 Once you have received tokens to an address, your balance will update and you will be able to view the transaction in the transaction history. To send these tokens, press the "Send" button from the dashboard. Here you can fill in the address you want to send to, enter an amount, and hit "Send".
 
-[![send payment ledger](/img/learn/firefly/send_payment_ledger.png)](/img/learn/firefly/send_payment_ledger.png)
+![send payment ledger](/img/learn/firefly/send_payment_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:276]:
```suggestion
[![send payment ledger](/img/learn/firefly/send_payment_ledger.png)](/img/learn/firefly/send_payment_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 13]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Note: As with generating addresses, this is a security check to ensure that you are connected to the official, secure version of Firefly. It ensures that no malicious software can alter the contents of your transaction and ensures you are sending the right amount to the correct address.
 
-[![confirm remainder address ledger](/img/learn/firefly/confirm_remainder_address_ledger.png)](/img/learn/firefly/confirm_remainder_address_ledger.png)[![confirm transaction ledger](/img/learn/firefly/confirm_transaction_ledger.png)](/img/learn/firefly/confirm_transaction_ledger.png)[![dispaly amount ledger](/img/learn/firefly/dispaly_amount_ledger.png)](/img/learn/firefly/dispaly_amount_ledger.png)[![display accept ledger](/img/learn/firefly/display_accept_ledger.png)](/img/learn/firefly/display_accept_ledger.png)
+![confirm remainder address ledger](/img/learn/firefly/confirm_remainder_address_ledger.png)![confirm transaction ledger](/img/learn/firefly/confirm_transaction_ledger.png)![dispaly amount ledger](/img/learn/firefly/dispaly_amount_ledger.png)![display accept ledger](/img/learn/firefly/display_accept_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:282]:
```suggestion
[![confirm remainder address ledger](/img/learn/firefly/confirm_remainder_address_ledger.png)](/img/learn/firefly/confirm_remainder_address_ledger.png)[![confirm transaction ledger](/img/learn/firefly/confirm_transaction_ledger.png)](/img/learn/firefly/confirm_transaction_ledger.png)[![dispaly amount ledger](/img/learn/firefly/dispaly_amount_ledger.png)](/img/learn/firefly/dispaly_amount_ledger.png)[![display accept ledger](/img/learn/firefly/display_accept_ledger.png)](/img/learn/firefly/display_accept_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 14]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 ### Creating wallets and sending internal transfers
 
 Firefly makes it possible to organize your tokens into different wallets within your profile. These tokens are kept separate from one another. Technically speaking, they are stored on addresses generated on different sub-accounts on your Ledger. Press "Create" from the dashboard and choose a name to add another wallet.
 
-[![multiple wallet view ledger](/img/learn/firefly/multiple_wallet_view_ledger.png)](/img/learn/firefly/multiple_wallet_view_ledger.png)
+![multiple wallet view ledger](/img/learn/firefly/multiple_wallet_view_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:288]:
```suggestion
[![multiple wallet view ledger](/img/learn/firefly/multiple_wallet_view_ledger.png)](/img/learn/firefly/multiple_wallet_view_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 15]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 With multiple wallets, you can then send between them and split your tokens up to organize them however you like. You can select "Internal Transfer" from the Send view and select which wallet you would like to send to.
 
 Note: Your most recently created wallet must receive funds (i.e. not be empty) before you can create a new one. This ensures that your wallets can be more smoothly recovered if you need to restore your Ledger profile on a new device.
 
-[![choose send from wallet ledger](/img/learn/firefly/choose_send_from_wallet_ledger.png)](/img/learn/firefly/choose_send_from_wallet_ledger.png)
+![choose send from wallet ledger](/img/learn/firefly/choose_send_from_wallet_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:294]:
```suggestion
[![choose send from wallet ledger](/img/learn/firefly/choose_send_from_wallet_ledger.png)](/img/learn/firefly/choose_send_from_wallet_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 16]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 ### Restoring an existing Ledger profile
 
 If you need to restore an existing Firefly profile with a new device, or you somehow lose access to Firefly, you can plug in your Ledger and restore an existing profile in a few simple steps. To do so, select "Migrate or restore a wallet" in setup and then "I have a Firefly Ledger backup".
 
-[![setup a wallet1 ledger](/img/learn/firefly/setup_a_wallet_ledger.png)](/img/learn/firefly/setup_a_wallet_ledger.png)[![create a wallet1 ledger](/img/learn/firefly/create_a_wallet_ledger.png)](/img/learn/firefly/create_a_wallet_ledger.png)
+![setup a wallet1 ledger](/img/learn/firefly/setup_a_wallet_ledger.png)![create a wallet1 ledger](/img/learn/firefly/create_a_wallet_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:300]:
```suggestion
[![setup a wallet1 ledger](/img/learn/firefly/setup_a_wallet_ledger.png)](/img/learn/firefly/setup_a_wallet_ledger.png)[![create a wallet1 ledger](/img/learn/firefly/create_a_wallet_ledger.png)](/img/learn/firefly/create_a_wallet_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 17]
```diff
@@ -230,83 +229,83 @@ Note: If you are an existing IOTA user and secured IOTA with a Ledger before the
...
 
 Note: Backups of your transaction history for Ledger Nano users are not currently supported but will be added in a later version. Once a profile is restored, Firefly will sync your transaction history. It will always find your balance. In a future version, we will add support for retrieving your full transaction history from a node that stores the complete history of the IOTA Tangle.
 
 ### Adding a second Ledger device to Firefly
 
 It is possible to have more than one Ledger device with the same Firefly app. You can add another device by creating an additional profile from the login page. Firefly will always check that the right device is connected and you will receive an error message if you try to generate an address or send a transaction with the wrong device.
 
-[![ledger multiple profile view](/img/learn/firefly/ledger_multiple_profile_view.png)](/img/learn/firefly/ledger_multiple_profile_view.png)
+![ledger multiple profile view](/img/learn/firefly/ledger_multiple_profile_view.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:308]:
```suggestion
[![ledger multiple profile view](/img/learn/firefly/ledger_multiple_profile_view.png)](/img/learn/firefly/ledger_multiple_profile_view.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 18]
```diff
@@ -530,79 +529,81 @@ You should thoroughly verify them before confirming the transaction.
...
 
 ### Finding balances keeps stalling, why is this?
 
-If your Ledger device does not show “Generating address…” while finding balances then try disconnecting and reconnecting the device. In rare cases, where you used an old version of the legacy app you may need to uninstall and reinstall the legacy app. Return to Ledger Live and reinstall this application on your Ledger device and try again.
+If your Ledger device does not show “generating address…” while finding balances then try disconnecting and reconnecting the device. In rare cases, where you used an old version of the legacy app you may need to uninstall and reinstall the legacy app. Return to Ledger Live and reinstall this application on your Ledger device and try again.
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:578]:
```suggestion
If your Ledger device does not show “Generating address…” while finding balances then try disconnecting and reconnecting the device. In rare cases, where you used an old version of the legacy app you may need to uninstall and reinstall the legacy app. Return to Ledger Live and reinstall this application on your Ledger device and try again.
```

Comment (Dr-Electron) [reply docs/learn/wallets/firefly-wallet.md:578]: This is a sentence from the ledger display. So it is fine as it is

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Phyloiota — APPROVED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**